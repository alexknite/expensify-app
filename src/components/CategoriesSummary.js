import React from 'react';
import { Link } from 'react-router-dom';

export const CategoriesSummary = () => {

  return (
    <div className="page-header">
      <div className="content-container">
        <h1 className="page-header__title">Categories</h1>
        <div className="page-header__actions">
          <Link className="button button--action" to="/create/category">Add Category</Link>
        </div>
      </div>
    </div>
  );
};

export default CategoriesSummary;
