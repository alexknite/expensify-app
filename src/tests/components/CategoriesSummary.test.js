import React from 'react';
import { shallow } from 'enzyme';
import CategoriesSummary from '../../components/CategoriesSummary';

test('should render CategoriesSummary correctly', () => {
  const wrapper = shallow(<CategoriesSummary />);
  expect(wrapper).toMatchSnapshot();
});
