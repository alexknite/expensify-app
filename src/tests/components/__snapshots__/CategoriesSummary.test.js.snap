// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CategoriesSummary correctly 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Categories
    </h1>
    <div
      className="page-header__actions"
    >
      <Link
        className="button button--action"
        replace={false}
        to="/create/category"
      >
        Add Category
      </Link>
    </div>
  </div>
</div>
`;
